endaq/device/schemata/mide_manifest.xml

```diff
@@ -168,6 +168,7 @@
         <!-- Communication -->
         <MasterElement name="CommunicationWiFi_ESP32" id="0x4D70" mandatory="0" multiple="0" minver="1">Indicates the presence of an ESP32 Wi-Fi module.</MasterElement>
         <MasterElement name="CommunicationNoSerial" id="0x4D71" mandatory="0" multiple="0" minver="1">Indicates device should not appear as a USB serial communication device, and accept file commands instead. </MasterElement>
+        <MasterElement name="CommunicationCorBus" id="0x4D72" mandatory="0" multiple="0" minver="1">Indicates device has a CorBus interface and will respond to/send CorBus commands. </MasterElement>
```

Comment:
Tiny nitpick: Can we call it `CommunicationCorbus`? I know we capitalize both the W and F in `CommunicationWiFi_ESP32`, but `Fi` doesn't mean anything alone, while `Bus` does.

Reply:
Yeah I can do that, I had it as `CorBus` originally since that seems to be the format all our repos/documentation uses

Reply:
It honestly doesn't make a difference; the arguments for and against are evenly matched. While my personal taste favors `CommunicationCorbus`, I'm approving as-is, and you can decide to change it or not.

Reply:
Thanks! I think I'll opt to be consistent with repos/documentation and leave it as `CommunicationCorBus`